elastic/kibana
# [AO] Fix APM latency expected value

Fixes #154801

## 📝 Summary 
This PR fixes the expected value of the APM latency rule both in the flyout and alert details page.

|Before|After 1|After 2|
|---|---|---|
|![image](https://user-images.githubusercontent.com/12370520/235477575-bc1891b7-0067-4bdf-9576-f0dce41d4b4c.png)|![image](https://user-images.githubusercontent.com/12370520/235477607-ab37d990-3536-409a-980d-370473a8c918.png)|![image](https://user-images.githubusercontent.com/12370520/235477667-438ebd69-3e4d-4e95-88f1-5fc8febf9125.png)|


## 🧪 How to test
- Create an APM Latency alert
- Click on the reason to open the flyout and check the expected value there
- From the action menu of that alert, go to alert details page and check the expected value
